test/test_pid_reuse.c

```diff
@@ -67,6 +67,11 @@ void* comp_file_multith (void *iterations)
 	sleep(sleep_sec[0]);
 
 	fprintf(stderr, "thread %d exits\n", (int) my_pid);
+	err = deflateEnd(&c_stream);
+	if (err != 0) {
+		printf("deflateEnd err %d\n", err);
+		return (void *) 1;
+	}
```

Comment:
Why are you adding this here just to remove on next commit?

Reply:
Strange, I still se this commit in the PR and in the https://github.com/lucmaga/power-gzip/tree/valgrind-work branch, but here at the top page the code is marked as Outdated.

Reply:
I will try to force push again..